pkg/meta/sql_test.go

```diff
@@ -17,13 +17,13 @@
 package meta
 
 import (
-	"os"
 	"testing"
+
+	"github.com/juicedata/juicefs/pkg/utils"
 )
 
 func TestSQLClient(t *testing.T) {
-	os.Remove("test.db")
-	m, err := newSQLMeta("sqlite3", "test.db", &Config{})
+	m, err := newSQLMeta("sqlite3", utils.RandString(8)+".db", &Config{})
```

Comment:
Should we use `io/ioutil.TempFile()`?

Reply:
+1, it seems to be a good use case for `TempFile`.

Reply:
e326154